site/public/css/plagiarism.css

```diff
@@ -97,14 +97,17 @@ input[type=text]:invalid {
     margin-right: 0;
 }
 
-#ignore_submission_div {
-    position: relative;
-    padding-bottom: 30px;
+#current_code_file, #provided_code_file, #prev_gradeable_div {
+    display: none;
 }
 
-#text_ignore_submission, #current_code_file, #provided_code_file,
-#add_more_ignore, #prev_gradeable_div {
-    display: none;
+#ignore_others_list {
```

Comment:
Submitty now mandates that IDs have words separated by dashes.  This file is currently ignored by the CSS linter but it's close to being able to be removed from the ignored list.  Can this be changed to be #ignore-others-list?

Reply:
Yup! looking at the twig file and the css, there are many other instances where we are using underscores instead of dashes in IDs. I'll fix that.